content/200-concepts/050-overview/500-under-the-hood.mdx

```diff
@@ -105,24 +116,24 @@ The output shows that the query engine path comes from the `PRISMA_QUERY_ENGINE_
 
    <tab>
 
-   ```sh highlight=2;normal
-   Current platform     : darwin
-   Query Engine         : query-engine d6ff7119649922b84e413b3b69660e2f49e2ddf3 (at /custom/my-query-engine-unix)
-   Migration Engine     : migration-engine-cli d6ff7119649922b84e413b3b69660e2f49e2ddf3 (at /myproject/node_modules/@prisma/engines/migration-engine-unix)
-   Introspection Engine : introspection-core d6ff7119649922b84e413b3b69660e2f49e2ddf3 (at /myproject/node_modules/@prisma/engines/introspection-engine-unix)
-   Format Binary        : prisma-fmt d6ff7119649922b84e413b3b69660e2f49e2ddf3 (at /myproject/node_modules/@prisma/engines/prisma-fmt-unix)
-   ```
+```sh highlight=2;normal
```

Comment:
Is this intentional?

Reply:
Not intentional. Must have been prettier formatting it